routes/opportunity.js

```diff
@@ -236,4 +236,71 @@ router.patch('/:opportunity_id', (req, res) => {
   opportunityController.updateOpportunity(req, res);
 });
 
+/**
+ * @openapi1
```

Comment:
There is a extra `1` after @openapi, due to which this API doesn't show up in Swagger UI at http://localhost:3030/playground